site/en/docs/extensions/reference/tabs/index.md

```diff
@@ -242,6 +242,19 @@ populates chrome.runtime.lastError is not unchecked.
   </web-tab>
 </web-tabs>
 
+### Pass a message to a selected tab's content script {: #messaging }
+
+This example demonstrates how an extension's service worker can communicate with content scripts in specific browser tabs using `tabs.sendMessage`.
+
+```js
+function sendMessageToActiveTab(messageInfo) {
```

Comment:
Thoughts on replacing `messageInfo` here and below with just `message`? I think that makes more sense because it is the message, it's not info about it.